screenpy_selenium/actions/hold_down.py

```diff
@@ -46,8 +44,10 @@ class HoldDown:
     @classmethod
     def command_or_control_key(cls: Type[SelfHoldDown]) -> SelfHoldDown:
         """
-        A convenience method that figures out what operating system the Actor
-        is using and directs the Actor which execution key to hold down.
+        A convenience method.
```

Comment:
I think this should be a little more descriptive, how about:

> A convenience method for supporting multiple operating systems.

"Operating systems" could be "OSs" if it's too long still.